src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Telemetry/IVsTelemetryService.cs

```diff
@@ -7,14 +7,9 @@ namespace Microsoft.VisualStudio.Telemetry
     /// <summary>
     /// Internal as we don't want anyone who depends on us to post events to our eventId.
     /// </summary>
-    internal interface ITelemetryService
+    internal interface IVsTelemetryService
     {
         /// <summary>
-        /// Posts a given telemetry event path to the telemetry service session for the program.
-        /// </summary>
-        void PostEvent(string eventName);
-
-        /// <summary>
         /// Posts a given telemetry operation to the telemetry service. <seealso cref="TelemetrySessionExtensions.PostOperation(TelemetrySession, string, TelemetryResult, string, TelemetryEventCorrelation[])"/>
         /// </summary>
         /// <param name="eventName">Name of the event.</param>
```

Comment:
The rest of interface is not needed in the managed layer? What if I want to post an operation from the managed layer? I guess my question is why do we need this interface? Why can't these methods be just implemented by the vstelemetryservice given all other telemetry code should just talk through the ITelemetryService?